nixos/modules/services/networking/picosnitch.nix

```
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.picosnitch;
+in
+{
+  options.services.picosnitch = {
+    enable = mkEnableOption (lib.mdDoc "picosnitch daemon");
+  };
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ pkgs.picosnitch ];
+    systemd.services.picosnitch = {
+      description = "picosnitch";
+      wantedBy = [ "multi-user.target" ];
+      serviceConfig = {
+        Type = "simple";
+        Restart = "always";
+        RestartSec = 5;
+        ExecStart = "${pkgs.picosnitch}/bin/picosnitch start-no-daemon";
+        PIDFile = "/run/picosnitch.pid";
```

```suggestion
        PIDFile = "/run/picosnitch/picosnitch.pid";
```
nixos usually puts them into their own directory

---

I already made that change https://github.com/NixOS/nixpkgs/pull/220761/commits/2e60a7dc5c33a0019d5c3398e9a673a56871683b